classes/Module/AddonsClientInterface.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\AutoUpgrade\Module;
```

Comment:
```suggestion
namespace PrestaShop\Module\AutoUpgrade\Addons;
```

Reply:
The thing is not all the classes in this namespace are about Addons
The ModuleDisabler is about modules in PrestaShop, doesn't matter if they come from Addons or not, so the namespace `Module` seemed more appropriate

Reply:
Oooh, I just got it. Maybe you suggest adding another `Addons` namespace, in addition to the `Module` namespace?

Reply:
Yep, One for `Addons` and one for `Module`  :sweat_smile: